src/libs/vtools/dialogs/tools/anchorpoint_dialog.h

```diff
@@ -49,26 +49,48 @@
  **
  *************************************************************************/
 
-#ifndef VISTOOLPIN_H
-#define VISTOOLPIN_H
+#ifndef DIALOGPIN_H
```

Comment:
Did you want to keep "pin" in DIALOGPIN_H, or did you want to change it to DIALOGANCHOR_H?
Just checking....